extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/augmented/AugmentedCacheInvalidateAllInterceptor.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.cache.runtime.augmented;
+
+import java.lang.annotation.Annotation;
+
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.cache.runtime.Cache;
+
+@AugmentedCacheInvalidateAll
+@Interceptor
+public class AugmentedCacheInvalidateAllInterceptor extends AugmentedCacheAnnotationInterceptor {
```

Comment:
I wonder if we should set a priority for these interceptors. @mkouba WDYT?

Reply:
It depends. Right now, an interceptor without priority has a priority `0` which corresponds to the `javax.interceptor.Interceptor.Priority.PLATFORM_BEFORE` value which is probably reasonable in this particular case?

This reminds me that this part of interceptor enablement is non-standard and undocumented ;-). I'll create a new issue.

Reply:
Anyway, it would make sense to add the `@Priority` annotation to be spec-compliant. Also the default behavior may change in the near future - see https://github.com/quarkusio/quarkus/issues/3767.

Reply:
Thanks for reviewing this @mkouba! I added the `@Priority` annotation with a `PLATFORM_BEFORE` value to all interceptors.